presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -599,6 +599,11 @@ default void dropMaterializedView(ConnectorSession session, SchemaTableName view
     /**
      * Get the materialized view status to inform the engine how much data has been materialized in the view
      */
+    default MaterializedViewStatus getMaterializedViewStatus(ConnectorSession session, SchemaTableName materializedViewName, TupleDomain<String> baseQueryDomain)
```

Comment:
A few comments here:
- Just realized that we use `String` in `MaterializedViewStatus` to represent a column. That probably is not ideal. We should use `ColumnHandle` and call them `columns` instead of `columnNames`. (Note that you can still compare between `ColumnHandle`s). If you have time, you can refactor the code to fix the issue in another PR.
- Please add java doc for this interface to explain the use of `baseQueryDomain`. That is important.

Reply:
Thanks for the feedback! I'll put the former in my to-do, seems like a pretty small ask I can do before I leave. If not I'll file an issue. Added the javadoc, let me know if you think it's descriptive.